cwms-radar-model/src/main/java/mil/army/usace/hec/cwms/radar/client/model/AssignedLocation.java

```diff
@@ -0,0 +1,57 @@
+package mil.army.usace.hec.cwms.radar.client.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.annotations.ApiModelProperty;
+
+@javax.annotation.Generated(value = "io.swagger.codegen.v3.generators.java.SpringCodegen", date = "2021-11-02T12:25:34.578-07:00[America/Los_Angeles]")
+public class AssignedLocation {
```

Comment:
Are these all of the attributes supported by the DB table?

Reply:
Yes